django/contrib/admin/static/admin/js/admin/DateTimeShortcuts.js

```diff
@@ -105,11 +105,21 @@
             shortcuts_span.className = DateTimeShortcuts.shortCutsClass;
             inp.parentNode.insertBefore(shortcuts_span, inp.nextSibling);
             var now_link = document.createElement('a');
-            now_link.setAttribute('href', "javascript:DateTimeShortcuts.handleClockQuicklink(" + num + ", -1);");
+            now_link.setAttribute('href', "#");
             now_link.appendChild(document.createTextNode(gettext('Now')));
+            addEvent(now_link, 'click', function(e) {
+                e.preventDefault();
+                DateTimeShortcuts.handleClockQuicklink(num, -1);
+            });
             var clock_link = document.createElement('a');
-            clock_link.setAttribute('href', 'javascript:DateTimeShortcuts.openClock(' + num + ');');
+            clock_link.setAttribute('href', '#');
             clock_link.id = DateTimeShortcuts.clockLinkName + num;
+            addEvent(clock_link, 'click', function(e) {
+                e.preventDefault();
+                e.stopPropagation();
```

Comment:
My JS is weak, but why is "stopPropagation" need here and not for the `now_link` too?